src/app/pages/sharing/smb/smb-session-list/smb-session-list.component.ts

```diff
@@ -46,14 +47,27 @@ export class SmbSessionListComponent implements OnInit {
     private ws: WebSocketService,
     private translate: TranslateService,
     private cdr: ChangeDetectorRef,
+    private route: ActivatedRoute,
     protected emptyService: EmptyService,
   ) {}
 
   ngOnInit(): void {
     const smbStatus$ = this.ws.call('smb.status', [SmbInfoLevel.Sessions]).pipe(
-      tap((sessions) => this.sessions = sessions),
+      tap((sessions) => {
+        this.sessions = sessions;
+        if (this.filterString) {
+          this.onListFiltered(this.filterString);
+        }
+      }),
       untilDestroyed(this),
     );
+
+    this.route.queryParams.pipe(untilDestroyed(this)).subscribe(params => {
+      if (params['s']) {
```

Comment:
> Is it used somewhere?

no, it's just a similar feature we have on the Dataset -> Snapshots page, I added it with a searching fix.
But I think we can remove it since no cases where it will be used.